build-test-local.xml

```diff
@@ -306,6 +306,7 @@ local.bundles.backup.file=${liferay.home.parent.dir}/bundles-master-${git.commit
 	</target>
 
 	<target name="setup-custom-suite">
+		<property name="integration" value="false" />
```

Comment:
One more, there's a space inbetween `"false"` and `/>`

Reply:
Just to be sure, this space is even in the other PR:
https://github.com/joshitagaki/liferay-portal-ee/pull/6/commits/fefc844823efa563deaff811c4bb19ee1301bd7a

Its an issue for both?

Reply:
At the line 312 we have `<isset property="custom.suite" />` there is space even here

Reply:
Oh that's true I think it's something else actually...